.changeset/lovely-pianos-build.md

```diff
@@ -0,0 +1,22 @@
+---
+'astro': minor
+---
+
+Provides a new, experimental build cache for [Content Collections](https://docs.astro.build/en/guides/content-collections/). This includes multiple refactors to Astro's build process to optimize how Content Collections are handled, which should provide significant performance improvements for users with many collections.
+
+You can opt in to preview the new build cache by adding the following flag to your Astro config:
+
+```js
+// astro.config.mjs
+export default {
+  experimental: {
+    contentCollectionCache: true,
+  },
+};
+```
+
+When this experimental feature is enabled, you can bypass the local build cache by passing the `--force` flag to `astro build`.
```

Comment:
Can you add some of the numbers to this? This will go into the article so it would be good for context.

Even specifically the docs numbers would be good as that is a real world use-case.

Reply:
Separately, would be good to mention a little about how this works as far as where the cache is stored, explaining how it is retained in CI, etc.

Reply:
Good call! Updated!